package.json

```diff
@@ -52,8 +52,8 @@
     "postinstall": "husky"
   },
   "lint-staged": {
-    "*": "prettier --write --cache --ignore-unknown",
-    "*.{ts,tsx,js,jsx}": "eslint --cache --fix"
+    "*": "pnpm exec prettier --write --cache --ignore-unknown",
+    "!(examples/**/*).{ts,tsx,js,jsx}": "pnpm exec eslint --cache --fix"
```

Comment:
Example folder has already been configured ignored in the eslint config file but not in lint-staged. So eslint will still check the folder when you try to commit change but it will always fail because of some plugins' missing.